KsApi/models/Project.swift

```diff
@@ -8,6 +8,7 @@ public struct Project {
   public var category: Category
   public var country: Country
   public var creator: User
+  public var graphQLProject: GraphQLProject?
```

Comment:
Yes, that's confusing. I renamed `graphQLProject` to `extendedProjectProperties` everywhere its' used in the app.
The reason I'm including this property inside the original project is because its' properties only come from GQL. So the endpoint we refactored for the project page only returns properties from GQL now but it is still using the original `Project` model as a return value. That way the application logic and tests do not need to change at all. Which is why its' logical to add this new property into the original v1 `Project` without creating a new call to return only those properties separately. The ultimate aim is to move all the `Project` properties into the `ExtendedProjectProperties` model and then rename it `Project`.

Let me know if you have another way to think about this, I definitely open to other approaches. Right now just trying to make it as easy and quick as possible to implement.

Reply:
Gotcha. As long as we aren't introducing any technical debt or risk down the line I'm confident in your decisions.